dietpi/dietpi-software

```diff
@@ -10385,7 +10392,27 @@ _EOF_
 			fi
 
 		fi
+#---------------------------------
+		software_id=142 # microk8s
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# APT deps
+			G_AGI snapd
+		#install
+		G_EXEC snap install microk8s --classic
+		# Enable cgroups used by Docker and alike
+		if [[ -f '/boot/boot.cmd' ]] && grep -q 'docker_optimizations' /boot/boot.cmd
+			then
+				# Armbian
+				[[ -f '/boot/armbianEnv.txt' ]] && G_CONFIG_INJECT 'docker_optimizations=' 'docker_optimizations=on' /boot/armbianEnv.txt
+				# Radxa Zero
+				[[ -f '/boot/uEnv.txt' ]] && G_CONFIG_INJECT 'docker_optimizations=' 'docker_optimizations=on' /boot/uEnv.txt
+			fi
 
+		fi
```

Comment:
Check out the updated cgroup handling for K3s above, which applies them on RPi and the new Odroid N2 image (and more of this type to come) as well. I think we should make this a dedicated function, like `Enable_memory_cgroup()` to avoid repeating code.